src/libraries/Microsoft.Extensions.Options/ref/Microsoft.Extensions.Options.csproj

```diff
@@ -20,4 +20,8 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="System.ComponentModel.Annotations" Version="$(SystemComponentModelAnnotationsVersion)" />
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
Probably this Condition is contributing to the build error?

Reply:
No, the failures are on Linux build. I am trying to figure it out.